WWCodeDFW/WWCodeDFW/ApiService.swift

```diff
@@ -15,6 +15,8 @@ enum ApiServiceError: Error {
     case failureRequestError
 }
 
+typealias ApiResult<T> = Result<T, Error>
```

Comment:
This should be `Result<T, ApiServiceError>` since all of `ApiResult`'s uses have en error type of `ApiServiceError`.

Reply:
My bad. I just fixed it